pkg/controller/deadmanssnitchintegration/deadmanssnitchintegration_controller.go

```diff
@@ -331,6 +331,11 @@ func (r *ReconcileDeadmansSnitchIntegration) createSnitch(dmsi *deadmanssnitchv1
 			snitch = snitches[0]
 		} else {
 			newSnitch := dmsclient.NewSnitch(snitchName, dmsi.Spec.Tags, "15_minute", "basic")
+			notes := fmt.Sprintf(`
```

Comment:
Does this not produce a needles empty line at the start?

Reply:
if it starts with (`cluster_id.., the formatting goes odd

Reply:
If it works as it is, then I guess leave it :)